extensions/amp-embedly/0.1/test/test-amp-embedly.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// import {AmpEmbedly} from '../amp-embedly';
+
+describes.realWin('amp-embedly', {
+  amp: {
+    extensions: ['amp-embedly'],
+  },
+}, env => {
+
+  let win;
+  let element;
+
+  beforeEach(() => {
+    win = env.win;
+    element = win.document.createElement('amp-embedly');
+    win.document.body.appendChild(element);
+  });
+
+  it('should have hello world when built', () => {
+    element.build();
+    expect(element.querySelector('div').textContent).to.equal('hello world');
```

Comment:
This doesn't test anything in the component, unfortunately.

Consider testing units here. (Psst, if you split into methods (as I suggested) and make them `@visibleForTesting` your life will be a lot easier).